Microsoft.Toolkit.Win32.UI.XamlApplication.Package/Microsoft.Toolkit.Win32.UI.XamlApplication.Package.csproj

```diff
@@ -7,7 +7,7 @@
     <Description>This library provides XAML islands common helpers for native applications. It is part of the Windows Community Toolkit.</Description>
     <PackageTags>XAML Islands XAMLHost</PackageTags>
     <PackageId>Microsoft.Toolkit.Win32.UI.XamlApplication</PackageId>
-    <Platforms>x64;x86;ARM64;AnyCPU</Platforms>
+    <Platforms>x64;x86;ARM;ARM64;AnyCPU</Platforms>
```

Comment:
This should be only AnyCPU as this is a NuGet package only.

Reply:
Let's fix this on another PR.